app/javascript/components/MainContainer/MainContainer.js

```diff
@@ -3,79 +3,88 @@ import RestaurantsList from "./RestaurantsList/RestaurantsList";
 import Filters from "./Filters/Filters"
 import Map from "./Map/Map"
 import './main_container.css'
+import { connect } from "react-redux";
+import { restListResponse, restChangeFilter } from "../actions/restaurantsActions";
+
 const axios = require('axios');
 
 
+@connect((store) => {
+    return {
+        restaurants: store.restaurantsList
+    };
+})
```

Comment:
While this is fine, we stopped using decorators in most of our code because its proposal changed or something bureaucratic like that.

Reply:
too bad
I struggled with making this decorator work for a few hours